code/parse/sexp.cpp

```diff
@@ -11981,33 +11982,43 @@ void sexp_cap_waypoint_speed(int n)
 
 /**
  * Causes a ship to jettison its cargo
- * note that the 2nd arg (jettison delay) is not implemented
  */
-void sexp_jettison_cargo(int n)
+void sexp_jettison_cargo(int n, bool jettison_new)
 {
 	char *shipname;
 	int ship_index;
+	int jettison_delay_or_speed;
```

Comment:
It doesn't *need* to be passed as a float, but using a float preserves more options for setting the speed, as well as being consistent with the rest of the code.  As for why it's an int here, the only numbers that can be used with sexps are ints.